kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties

```diff
@@ -14,3 +14,5 @@ loginAsAnotherUser=Connectez-vous en tant qu'un autre utilisateur
 logoutSuccssful=D\u00e9connexion r\u00e9ussie
 # translation auto-copied from project KIE webapp distributions, version 6.2.0, document org.kie/kie-wb-webapp/org/kie/workbench/client/resources/i18n/LoginConstants, author croe@redhat.com
 loginAgain=Connectez-vous \u00e0 nouveau
+loadingPleaseWait=Veuillez patienter
+loadingApplication=Chargement de l'application...
```

Comment:
Correct... Sorry @mbiarnes for not spotting this!

We need [this](https://github.com/droolsjbpm/kie-wb-distributions/commit/4c82d6831773fff0b84cd36665b90c4cc9b3596b) applying to 6.5.x too.